specification/schema/hugr_schema_v0.json

```diff
@@ -0,0 +1,1726 @@
+                    "type": "integer"
+                },
+                "input_extensions": {
+                    "items": {
+                        "type": "string"
+                    },
+                    "title": "Input Extensions",
+                    "type": "array"
+                },
+                "op": {
+                    "const": "Input",
+                    "default": "Input",
+                    "title": "Op"
+                },
+                "types": {
+                    "items": {
+                        "$ref": "#/$defs/Type"
+                    },
+                    "title": "Types",
+                    "type": "array"
+                }
+            },
+            "required": [
+                "parent"
+            ],
+            "title": "Input",
+            "type": "object"
+        },
+        "Int": {
+            "description": "An arbitrary size integer.",
+            "properties": {
+                "t": {
+                    "const": "I",
+                    "default": "I",
+                    "title": "T"
+                },
+                "width": {
+                    "title": "Width",
+                    "type": "integer"
+                }
+            },
+            "required": [
+                "width"
+            ],
+            "title": "Int",
+            "type": "object"
+        },
+        "LeafOp": {
+            "discriminator": {
+                "mapping": {
+                    "CustomOp": "#/$defs/CustomOp",
+                    "MakeTuple": "#/$defs/MakeTuple",
+                    "Noop": "#/$defs/Noop",
+                    "Tag": "#/$defs/Tag",
+                    "TypeApply": "#/$defs/TypeApply",
+                    "UnpackTuple": "#/$defs/UnpackTuple"
+                },
+                "propertyName": "lop"
+            },
+            "oneOf": [
+                {
+                    "$ref": "#/$defs/CustomOp"
+                },
+                {
+                    "$ref": "#/$defs/Noop"
+                },
+                {
+                    "$ref": "#/$defs/MakeTuple"
+                },
+                {
+                    "$ref": "#/$defs/UnpackTuple"
+                },
+                {
+                    "$ref": "#/$defs/Tag"
+                },
+                {
+                    "$ref": "#/$defs/TypeApply"
+                }
+            ]
+        },
+        "ListParam": {
+            "properties": {
+                "tp": {
+                    "const": "List",
+                    "default": "List",
+                    "title": "Tp"
+                },
+                "param": {
+                    "$ref": "#/$defs/TypeParam"
+                }
+            },
+            "required": [
+                "param"
+            ],
+            "title": "ListParam",
+            "type": "object"
+        },
+        "LoadConstant": {
+            "description": "Load a static constant in to the local dataflow graph.",
```

Comment:
Most of the other "descriptions" are noun phrases but this is a verb. Suggest "An operation that loads"?